README.md

```diff
@@ -1,24 +1,28 @@
-# Homebrew
-Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).
+# **Homebrew**
+**Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).**
 
 ## What Packages Are Available?
-1. Type `brew search` for a list.
-2. Or visit [braumeister.org](http://braumeister.org) to browse packages online.
-3. Or use `brew search --desc <keyword>` to browse packages from the command line.
+There are 3 ways you can find packages:
++ Use `brew search` in the command line for a list of packages.
++ Use `brew search --desc <keyword>` to browse packages from the command line.
++ Alternatively, you can visit [braumeister.org](http://braumeister.org) to browse packages online.
```

Comment:
I'm sorry, but this just makes the section longer without actually adding value. There are also more than the listed three ways to find packages and the list isn't intended to be exhaustive (it's just a starting point for new users).

However, I agree with most of the wording changes and the different order of the items. Note that it's “on the command line”, not “in”. (We can discuss other details once this section has been reverted to a numbered list without an introductory line.)